po/CMakeLists.txt

```diff
@@ -56,40 +57,56 @@ if(ENABLE_POT_UPDATE_TARGET)
 		# Update the source file dependencies.
 		include(update_pot_source_dependencies)
 
-		# Generate pot file for c++ data.
+		# Generate pot file for C++ strings.
 		add_custom_command(
 			# misses bug address
 			OUTPUT ${PROJECT_SOURCE_DIR}/po/${DOMAIN}/${DOMAIN}.cpp.pot
 			# create the pot file make sure we always get output.
 			COMMAND ${GETTEXT_XGETTEXT_EXECUTABLE} ${GETTEXT_XGETTEXT_OPTIONS}
-					--files-from=${PROJECT_SOURCE_DIR}/po/${DOMAIN}/POTFILES.in
+					--files-from=${PROJECT_SOURCE_DIR}/po/${DOMAIN}/POTFILES_CPP.in
 					--output=${PROJECT_SOURCE_DIR}/po/${DOMAIN}/${DOMAIN}.cpp.pot
 			# replace the chartype
 			COMMAND sed -i
 				s/charset=CHARSET/charset=UTF-8/
 				${PROJECT_SOURCE_DIR}/po/${DOMAIN}/${DOMAIN}.cpp.pot
-			# Remove some header info - Need to test whether needed.
-			DEPENDS ${PROJECT_SOURCE_DIR}/po/${DOMAIN}/POTFILES.in
+			DEPENDS ${PROJECT_SOURCE_DIR}/po/${DOMAIN}/POTFILES_CPP.in
 			WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
-			COMMENT "pot-update [${DOMAIN}]: Generated source pot file."
+			COMMENT "pot-update [${DOMAIN}]: Generating source pot file."
 		)
 
-		# Generate pot file for wml data.
+		# Generate pot file for Python strings.
+		add_custom_command(
+			OUTPUT ${PROJECT_SOURCE_DIR}/po/${DOMAIN}/${DOMAIN}.py.pot
+			# create the pot file make sure we always get output.
+			COMMAND ${GETTEXT_XGETTEXT_EXECUTABLE} ${GETTEXT_XGETTEXT_OPTIONS}
+					--language=Python
+					--files-from=${PROJECT_SOURCE_DIR}/po/${DOMAIN}/POTFILES_PY.in
+					--output=${PROJECT_SOURCE_DIR}/po/${DOMAIN}/${DOMAIN}.py.pot
+			# replace the chartype
+			COMMAND sed -i
+				s/charset=CHARSET/charset=UTF-8/
+				${PROJECT_SOURCE_DIR}/po/${DOMAIN}/${DOMAIN}.py.pot
+			DEPENDS ${PROJECT_SOURCE_DIR}/po/${DOMAIN}/POTFILES_PY.in
+			WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
+			COMMENT "pot-update [${DOMAIN}]: Generating python pot file."
+		)
+		# Generate pot file for WML/Lua strings.
 		add_custom_command(
 			OUTPUT ${PROJECT_SOURCE_DIR}/po/${DOMAIN}/${DOMAIN}.wml.pot
 
-			COMMAND ${WMLXGETTEXT}
-				--directory=${PROJECT_SOURCE_DIR}
+			COMMAND ${Python_EXECUTABLE} ${WMLXGETTEXT}
+				--force-po
 				--domain=${DOMAIN}
+				--directory=${PROJECT_SOURCE_DIR}/data
```

Comment:
This path includes `data`, however the paths outputted by the `FINDCFG` scripts also include `data`. After removing this line completely (so it uses the existing `WORKING_DIRECTORY` line), the data seems good for CMake.

> Hmmm. That's not [what I see](https://github.com/wesnoth/wesnoth/compare/master...Slayer95:wesnoth:wmltools-gui-i18n-test?expand=1).

The "cmake in Windows" commit in that branch changes the next couple of lines to use `--recursive` instead of FINDCFG.

Reply:
Oh... this diff was a leftover from that branch, actually.